manifests/service.pp

```diff
@@ -42,4 +34,11 @@
       enable => $foreman_service_enable,
     }
   }
+
+  foreman::dynflow::worker { 'orchestrator':
```

Comment:
Perhaps this should also be wrapped in `if $::foreman::jobs_manage_service {` rather than at the worker level.